client/spec/components/ProjectDetail.spec.jsx

```diff
@@ -72,9 +73,10 @@ describe( "ProjectDetail Component", () => {
 					owners: [],
 					versions: {}
 				} );
-				component.setProps( {
-					params: { name: "new-project", owner: "new-owner", branch: "branch" }
-				} );
+				ReactDOM.render(
```

Comment:
Although not necessary, it may be clearer if this line saves off the return value back into the `component` variable that is used in each subsequent `it` test case (like line number 32).